tests/integration/test_lists/qa/examples_test_list.txt

```diff
@@ -311,7 +315,10 @@ examples/test_medusa.py::test_llm_medusa_1gpu[use_py_session-medusa-vicuna-7b-v1
 examples/test_medusa.py::test_llm_medusa_fp8_modelOpt_ckpt_1gpu[use_cpp_session-llama3.1-medusa-8b-hf_v0.1-bs1]
 examples/test_medusa.py::test_llm_medusa_fp8_modelOpt_ckpt_1gpu[use_py_session-llama3.1-medusa-8b-hf_v0.1-bs8]
 examples/test_mistral.py::test_llm_mistral_lora_1gpu[komt-mistral-7b-v1-lora-komt-mistral-7b-v1]
-examples/test_mistral.py::test_llm_mistral_quantization_8gpus_summary[mistral-7b-v0.1-fp8-nb:4]
+examples/test_mistral.py::test_llm_mistral_quantization_summary[mistral-7b-v0.1-tp4pp2-nb:4]
+examples/test_mistral.py::test_llm_mistral_quantization_summary[Codestral-22B-v0.1-tp1pp1-nb:1]
+examples/test_mistral.py::test_llm_mistral_quantization_summary[Ministral-8B-Instruct-2410-tp1pp1-nb:1]
+examples/test_mistral.py::test_llm_mistral_quantization_summary[Mistral-Small-24B-Base-2501-tp1pp1-nb:1]
```

Comment:
If Codestral/Ministral/Mistral-Small share the same model structure with mistral, we don't need to test quantization summary again.